pkg/logs/scheduler/scheduler.go

```diff
@@ -72,23 +155,6 @@ func (s *Scheduler) Schedule(configs []integration.Config) {
 			for _, source := range sources {
 				s.sources.AddSource(source)
 			}
-		case s.newService(config):
-			entityType, _, err := s.parseEntity(config.TaggerEntity)
-			if err != nil {
-				log.Warnf("Invalid service: %v", err)
-				continue
-			}
-			// logs only consider container services
-			if entityType != containers.ContainerEntityName {
-				continue
-			}
-			log.Infof("Received a new service: %v", config.Entity)
-			service, err := s.toService(config)
-			if err != nil {
-				log.Warnf("Invalid service: %v", err)
-				continue
-			}
-			s.services.AddService(service)
```

Comment:
@djmitche thanks to this, we can finally get rid of _[this hack](https://github.com/DataDog/datadog-agent/blob/7.32.x/pkg/autodiscovery/autoconfig.go#L640-L649)_ in autodiscovery. I think it should have been done in this PR.